src/robot_properties_nyu_finger/robot_properties_nyu_finger/__init__.py

```diff
@@ -0,0 +1,7 @@
+"""__init__
```

Comment:
if we are not using rviz, please put the resources files in ` src/robot_properties_nyu_finger/resources/`

Reply:
if needed I will figure something out for rviz later

Reply:
@jviereck  Do you plan to use rviz? Since this is for internal use only, I suggest we remove it.

Reply:
I don't plan on using rviz. If we need a visualization, I would use meshcat for it.